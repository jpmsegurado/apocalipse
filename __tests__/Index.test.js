import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer';
import Index from '../pages/index';


describe('With Enzyme', () => {
  it('Label shows "Usuário"', () => {
    const app = shallow(<Index />)

    expect(app.find('label').text()).toEqual('Usuário')
  })
})

describe('With Snapshot Testing', () => {
  it('Label shows "Usuário"', () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})