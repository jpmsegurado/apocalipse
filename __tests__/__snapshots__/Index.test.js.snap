// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing Label shows "Usuário" 1`] = `
<div
  className=""
>
  <link
    href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
    rel="stylesheet"
  />
  <link
    href="https://npmcdn.com/react-bootstrap-table/dist/react-bootstrap-table-all.min.css"
    rel="stylesheet"
  />
  <link
    href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
    rel="stylesheet"
  />
  <script
    src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAD3yTq68bvCM8xkUSaByG3LETyCQaRzmU"
  />
  <div
    className="col-xs-3 col-xs-offset-6 login"
    data-jsx={993884036}
  >
    <div
      className="jumbotron"
      data-jsx={993884036}
    >
      <form
        className="container"
        data-jsx={993884036}
        onSubmit={[Function]}
      >
        <div
          className="form-group"
          data-jsx={993884036}
        >
          <label
            data-jsx={993884036}
            htmlFor="id"
          >
            Usuário
          </label>
          <input
            className="form-control"
            data-jsx={993884036}
            id="id"
            name="id"
            onChange={[Function]}
            placeholder="Seu ID"
            type="text"
          />
        </div>
        <button
          className="btn btn-primary btn-block"
          data-jsx={993884036}
          disabled={undefined}
          type="submit"
        >
          Entrar
        </button>
      </form>
    </div>
  </div>
</div>
`;
